javamelody-spring-boot-starter/src/test/java/net/bull/javamelody/TestJavaMelodyAutoConfiguration.java

```diff
@@ -59,25 +59,23 @@
 		// nothing
 	}
 
-	/**
-	 * test.
-	 */
-	@SuppressWarnings("unchecked")
 	@Test
-	public void testJavaMelodyAutoConfigurationIsCreated() {
-		// It should create a session listener.
-		final ServletListenerRegistrationBean<EventListener> sessionListenerRegistrationBean = context
+	public void testJavaMelodyAutoConfigurationIsCreatedThenItShouldCreateASessionListener() {

```

Comment:
I could revert it but a test that test about 50 functionality is bad. I tried to split it according to comment, if comment are necessary to know what we want to test, we should split it. Personally, I would have split it more.